jobs/UnitTest/unit_test.sh

```diff
@@ -44,4 +45,5 @@ prepare_deps $1
 pushd ${WORKSPACE}/build-deps/$1
 unit_test $1
 cp test/$1.xml ${WORKSPACE}/xunit-reports
+sudo chown -R ${USER}:${USER} test coverage static
```

Comment:
this will fail other repo's PR Gate. like on-core, because they don't have static folder in their work-space,
so it will fail on-core's PR Gate !!!